src/v0/destinations/google_adwords_enhanced_conversions/transform.js

```diff
@@ -84,9 +84,14 @@ const responseBuilder = async (metadata, message, { Config }, payload) => {
     response.headers['login-customer-id'] = filteredLoginCustomerId;
   }
 
-  if (loginCustomerId) {
-    const filteredLoginCustomerId = removeHyphens(loginCustomerId);
-    response.headers['login-customer-id'] = filteredLoginCustomerId;
+  if (
+    (isDefined(Config.subAccount) && Config.subAccount === true) ||
```

Comment:
your new UI shouldn't depend on old UI field of `subAccount` but should depend on new UI field `configData`

Reply:
And raise PR to hotfix branch

Reply:
`{
                rudderAccountId: '25u5whFH7gVTnCiAjn4ykoCLGoC',
                listOfConversions: ['Page View', 'Product Added'],
                authStatus: 'active',
                subAccount: false,
                configData: '{"customerId": "1234567890", "loginCustomerId": "12345"}',
                customerId: '1234567890',
                loginCustomerId: '12345',
              },`
              
This is how the config will look for edge case when subAccount is flase but config has field loginCustomerId. ConfigData will have loginCustomerId but need to depend on subAccount if it to be used or not.
 
 for new connections, configData will be  {"customerId": "1234567890", "loginCustomerId": "12345"} and no key `subAccount` in Config so use login customerId

Reply:
In that case, if `configDetails` object is not null, you shouldn't execute old logic - right now code is messed up where both old & new flows will be executed.

Reply:
`{
                rudderAccountId: '25u5whFH7gVTnCiAjn4ykoCLGoC',
                listOfConversions: ['Page View', 'Product Added'],
                authStatus: 'active',
                subAccount: false,
                configData: '{"customerId": "1234567890", "loginCustomerId": "12345"}',
                customerId: '1234567890',
                loginCustomerId: '12345',
              },`
              
This is how the config will look for edge case when subAccount is flase but config has field loginCustomerId. ConfigData will have loginCustomerId but need to depend on subAccount if it to be used or not.
 
 for new connections, configData will be  {"customerId": "1234567890", "loginCustomerId": "12345"} and no key `subAccount` in Config so use login customerId

Reply:
`{
                rudderAccountId: '25u5whFH7gVTnCiAjn4ykoCLGoC',
                listOfConversions: ['Page View', 'Product Added'],
                authStatus: 'active',
                subAccount: false,
                configData: '{"customerId": "1234567890"}',
                customerId: '1234567890',
                loginCustomerId: '12345',
              },`
              
updated code, once migration is done will remove old code that is kept as to be backward compatible